### ISSUE TITLE
Material Export creates custom shader nodes

### ISSUE THREAD

**Issue by Drigax** [BabylonJS/Exporters#911]
Remove our functionality added via that creates custom shader nodes to contain Babylon material export flags. We should instead use custom properties or some other workflow that doesn't modify the scene.

**Comment by pandaGaume** [comment]
@rdurnin Did you plan to work on the topic soon ? some issues arise regarding the nodes and we need to know if this enhancement will be available soon, otherwise we continue to support the nodes and correct bug.
Many thanks for the support and wish u an happy new year.

**Comment by rdurnin** [comment]
Hey, and I have some ideas for how I would like to change the code, but
have not begun to implement them. If you have changes you are waiting to
fold in, please continue with them and I will push a branch for you to
review on my forked repo.

**Comment by pandaGaume** [comment]
hey @rdurnin , how are you, I wanted to know if you had had time to move forward on this subject and if not, could you explain to me what your ideas are to replace the nodes.
G.

**Comment by rdurnin** [comment]
Hey @pandaGaume , I am well, and I hope the same. I have been working for the last few months on a large project using Babylon, and have needed to use a variety of exporters and tools to access all of features I wanted to use. If there is interest, I would like to share the tools I created for the project with the community, and to discuss some ideas I have for the Maya exporter. I should be able to return to this work next week, after I have release a first version of the project, and please let me know when you have time to discuss.
R.

**Comment by Drigax** [comment]
Yeah, that sounds great @rdurnin feel free to post in the babylon.js forum when you are ready to share your tools and ideas
